store/indexed.go

```diff
@@ -596,22 +596,20 @@ func (s *indexedUnitStore) Import(data graph.Output) error {
 	var defOfs, refOfs byteOffsets
 	var refFBRs fileByteRanges
 
-	par := parallel.NewRun(2)
-	par.Do(func() (err error) {
-		defOfs, err = s.fsUnitStore.writeDefs(data.Defs)
-		return err
-	})
-	par.Do(func() (err error) {
-		refFBRs, refOfs, err = s.fsUnitStore.writeRefs(data.Refs)
+	//Complete each write operation serially which eliminates
+	//a lock contention bug on slower underlying file stores
```

Comment:
Style nitpick, Go comments should have a space between `//` and the content. Not having a space is used for executable directives, e.g., `//go:generate` and temporarily commented out code (which usually doesn't get committed), e.g., `//log.Println(someDebugValue)`.

You can see good examples of comments in [this file](https://sourcegraph.com/github.com/golang/go@master/-/tree/src/net/http/transfer.go), for example.